session_test.go

```diff
@@ -164,13 +164,20 @@ func TestSession_RatchetDecrypt_CommunicationSkippedMessages(t *testing.T) {
 		m1.Ciphertext[len(m1.Ciphertext)-1] ^= 10
 		_, err := bob.RatchetDecrypt(m1, nil) // Error: invalid signature.
 		require.NotNil(t, err)
-		require.EqualValues(t, 0, bob.MkSkipped.Count(bob.DHr))
+
+		bobSkippedCount, err := bob.MkSkipped.Count(bob.DHr)
+		require.NoError(t, err)
```

Comment:
Line spacing here is not consistent with same code in previous file and further down in this file.